src/components/views/rooms/RoomHeader.tsx

```diff
@@ -81,32 +78,11 @@ export default function RoomHeader({ room }: { room: Room }): JSX.Element {
         return SdkConfig.get("element_call").use_exclusively && groupCallsEnabled;
     }, [groupCallsEnabled]);
 
-    const placeCall = useCallback(
-        async (callType: CallType, platformCallType: typeof voiceCallType) => {
-            switch (platformCallType) {
-                case "legacy_or_jitsi":
-                    await LegacyCallHandler.instance.placeCall(room.roomId, callType);
-                    break;
-                // TODO: Remove the jitsi_or_element_call case and
-                // use the commented code below
-                case "element_call":
-                case "jitsi_or_element_call":
-                    defaultDispatcher.dispatch<ViewRoomPayload>({
-                        action: Action.ViewRoom,
-                        room_id: room.roomId,
-                        view_call: true,
-                        metricsTrigger: undefined,
-                    });
-                    break;
-
-                // case "jitsi_or_element_call":
-                // TODO: Open dropdown menu to choice between
-                // EC and Jitsi. Waiting on Compound's dropdown
-                // component
-                // break;
-            }
+    const makeCall = useCallback(
```

Comment:
This function can probably be removed and `placeCall` can be called directly.
I can't see any value in it except that it raises the complexity here.